model/registry/otel.yaml

```
@@ -1,4 +1,28 @@
 groups:
+  - id: registry.otel
+    prefix: otel
+    type: resource
+    brief: Attributes reserved for OpenTelemetry
+    attributes:
+      - id: status_code
+        type:
+          allow_custom_values: true
+          members:
+            - id: ok
+              value: OK
+              brief: 'The operation has been validated by an Application developer or Operator to have completed successfully.'
+              stability: experimental
```

```suggestion
              stability: stable
```
same for error, the attribute is stable

---

they were defined experimental before, was it a mistake? because the field itself is stable

---

right, sounds like it was  wrong before

---

Changed to stable